scipy/stats/tests/test_multivariate.py

```diff
@@ -260,17 +260,22 @@ def test_marginalization(self):
 
     def test_frozen(self):
         # The frozen distribution should agree with the regular one
+        rvs_seed = 4567
         np.random.seed(1234)
         x = np.random.randn(5)
         mean = np.random.randn(5)
         cov = np.abs(np.random.randn(5))
-        norm_frozen = multivariate_normal(mean, cov)
+        norm_frozen = multivariate_normal(mean, cov, seed=rvs_seed)
         assert_allclose(norm_frozen.pdf(x), multivariate_normal.pdf(x, mean, cov))
         assert_allclose(norm_frozen.logpdf(x),
                         multivariate_normal.logpdf(x, mean, cov))
         assert_allclose(norm_frozen.cdf(x), multivariate_normal.cdf(x, mean, cov))
         assert_allclose(norm_frozen.logcdf(x),
                         multivariate_normal.logcdf(x, mean, cov))
+        n = 2
+        assert_array_almost_equal(
+            norm_frozen.rvs(n), multivariate_normal.rvs(mean, cov, size=n,
+                                                        random_state=rvs_seed))
```

Comment:
Good test to check that nothing changes.

Reply:
If it's OK for the numbers to change, then maybe test against:
```
from scipy.stats import multivariate_normal
import numpy as np

cov = np.array([[4, 1], [1, 3]])
mean = np.array([1, 2])
size = (3, 4)
rng = np.random.default_rng(0)

# direct implementation of https://en.wikipedia.org/wiki/Multivariate_normal_distribution#Drawing_values_from_the_distribution
L, S = np.linalg.eigh(cov)
A = S * np.sqrt(L)
x = rng.normal(size=(*size, 2))
rvs0 = mean + x @ A.T

rng = np.random.default_rng(0)
rvs1 = multivariate_normal.rvs(size=size, mean=mean, cov=cov, random_state=rng)
```
If we have to bring this in line with NumPy, test against that.

Reply:
I've added a test to compare against numpy random variates.